packages/datetime/src/common/utils.ts

```diff
@@ -4,6 +4,14 @@
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
 
+function removeElement(childNode: Node) {
```

Comment:
import from core

Reply:
or just inline this in the one usage below. `x.parent.removeChild(x)`

Reply:
Are you sure? It seems strange for core to export this utility, and therefore promise support for that feature, when it's a pretty trivial implementation detail that's also cheap to reimplement.

Reply:
Ah, I like the inlining -- I'll do that. Thanks!